src/lib.rs

```diff
@@ -67,9 +67,15 @@ impl TryFrom<&str> for Organisationsnummer {
             None => 0,
         };
 
+        let mut number = org.to_string().replace("-", "");
```

Comment:
I've replaced both + and - here, not sure if it's really needed, but just to make sure it works with a personnummer for a person whom is 100+ and is a EF, which I'm super sure happens a lot ;P

Reply:
It's fine, i'll try the number with personnummer package before trying to parse the organization number and those numbers can't have `+`